recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -31,4 +46,30 @@ do_deploy:append:freedom-u540() {
     fi
 }
 
+do_configure[depends] += "opensbi:do_deploy"
```

Comment:
do we really need opensbi artifacts during configure step ? I wonder if it should be do_compile

Reply:
Not sure why freedom-u540 u-boot doesn't need to depend on OpenSBI. 
For AE350 platform, it needs OPENSBI path to be set to fw_dynamic.bin before binman generates u-boot.itb, 
it can be do_compile.

Reply:
Change applied, thanks.